src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
Also on the subject of naming things, the delete-all-the-data function I've been thinking of, any naming related preferences? deleteNodes? purgeNodes? Something else?